README.md

```diff
@@ -1,29 +1,91 @@
 # web3-login-demo
 
-This website demonstrates how to build a logging using KILT Protocol.
+_Explanatory Tutorial for websites to implement logging in with web3 credentials using the KILT Protocol._
 
-During the logging procedure, the user is required to present a credential.
+## Intro
+
+This project demonstrates how to enable **login using KILT Protocol**.
+It is an educational project, everything is broken down to little steps and has explanations to ease your understanding.
+
+During [the login procedure](#login-process), the user is required to present a credential.
 You can decide which credentials to accept.
 They could be issued by yourself or by other attesters.
-In this demo we request a credential that contains an email address that the user owns.
-For that we rely on https://socialkyc.io to issue email credentials.
+
+This is basically like asking for an ID-Card instead of a customer number, in order to provide a personalized service.
```

Comment:
Ok. Yes, I guess some people run away from the idea of showing their government ID-Card.

An Identification card could be anything though; like from a university, a club, a company, etc. Not necessarily a membership.  But is true that people first think of government credentials. Better to avoid it.



I will replace line 14 with your approach.

Reply:
done